quasar-core/src/main/java/co/paralleluniverse/common/util/ExtendedStackTrace.java

```diff
@@ -82,8 +82,9 @@ protected ExtendedStackTrace(Throwable t) {
         Member[] ms = getMethods(este.getDeclaringClass());
         Member method = null;
 
+        final String methodName = este.getMethodName();
```

Comment:
This is a very bland name for a variable. What is so special about this _particular_ method?

Reply:
We are going through the stack trace, looking at each method. `este` is a single entry in the stack trace and we are going to try to find the matching method. So `methodName` is what we are looking for.

Would you prefer `stackTraceMethodName`, `methodNameToFind`, `wantedMethodName` or something else? I could also inline this variable and go back to what we had before.

Reply:
So maybe `targetMethodName`?

Reply:
sounds good. I'll make the change now